activestorage/app/models/active_storage/attachment.rb

```diff
@@ -36,7 +36,13 @@ def identify_blob
     end
 
     def analyze_blob_later
-      blob.analyze_later unless blob.analyzed?
+      return if blob.analyzed?
+
+      if blob.analyzable?
+        blob.analyze_later
+      else
+        blob.analyze
```

Comment:
Don’t see the point of running analyze if the blob isn’t analyzable. Otherwise it shouldn’t be called analyzable because that’s not what it’s checking then.

Reply:
I was thinking from a data consistency perspective it would make sense to no-op analyze a blob that doesn't need analysis, but maybe not. I'll refactor to not run analysis at all and see what that looks like.

Reply:
Seems like the question we want to ask them is about deferring the analyze and we’d need some structure around that.

Maybe `analyze_later?` and put that in analyze_later?